guids-generator.html

```diff
@@ -456,17 +499,45 @@
                         />
                     </Grid>
                     <Grid item>
-                        <Button onClick={() => onSubmit()} variant="contained" color="primary" disabled={!(binput?.trim().length > 0)}>Generate GUIDs</Button>
+                      <Button onClick={() => onSubmit()} variant="contained" color="primary" disabled={!(binput?.trim().length > 0)}>Generate GUIDs</Button>
                     </Grid>
                 </Grid>
                 { Object.keys(bhashes).length > 0 &&
-                    <List>
-                        { validatedData.map( item =>
-                            <ListItem className={classes.listItem}>
-                                <ListItemText primary={bhashes[item.index].value} style={{color: bhashes[item.index].isError ? 'red' : 'black'}} />
-                            </ListItem>
-                        )}
-                    </List>
+                    <TableContainer component={Paper}>
+                        <Table className={classes.table} size="small">
+                            <TableHead>
+                              <TableRow>
+                                <StyledTableCell>Health card #</StyledTableCell>
+                                <StyledTableCell>Province</StyledTableCell>
+                                <StyledTableCell>Date of birth</StyledTableCell>
+                                <StyledTableCell>GUID
```

Comment:
There's still a slight alignment issue, GUID is higher than all other table head labels.

![image](https://user-images.githubusercontent.com/651980/96633806-a3377c80-12e7-11eb-8274-88d51564317c.png)